etc/profile-a-l/gtk-lbry-viewer.profile

```diff
@@ -6,7 +6,10 @@ include gtk-lbry-viewer.local
 # added by included profile
 #include globals.local
 
-ignore quiet
+private-bin gtk-lbry-viewer
+
+# Redirect
+include gtk-youtube-viewers-common.profile
```

Comment:
```suggestion
include gtk-youtube-viewers-common.profile
```

Leave a single redirect comment on each profile to make the includes a bit less
confusing.